i18n/es.jsonp.js

```diff
@@ -19,8 +19,14 @@ document.localeJson = {
       "configure-title": "Configuración",
       "configure-about-usage-link": "Información (Uso)",
       "configure-btn-library": "Biblioteca ZIM",
+      "configure-btn-folderselect": "Seleccione carpeta",
+      "configure-btn-rescan": "Volver a escanear",
+      "configure-about-rescan-tip": "Vuelve a escanear las tarjetas SD y la memoria interna",
+      "configure-select-file-numbers": "archivo(s) encontrados en la ubicación seleccionada. ",
       "configure-download-instructions": "Esta aplicación necesita un archivo ZIM para funcionar.<br />Para instrucciones completas, vea la sección",
       "configure-select-instructions": "Seleccione o arrastre y suelte un archivo .zim (o todos los .zimaa, .zimab etc en caso de un archivo dividido):",
+      "configure-select-file-instructions": "Seleccione el archivo que desea utilizar:",
+      "configure-select-file-first-option": "Selecciona un archivo..",
```

Comment:
I just realized this should be "Seleccione" (like the line above) instead of "Selecciona". (The latter is informal, which is not usually used in software or operating systems.) Also, you're missing one extra full stop to make ellipsis.